include/implem/trace.h

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
isn't this file already somewhere?

Reply:
If not, and we need only `ALLOC_TRACE`, why are we defining a whole set of new ones?